enterprise/resign/resignOwncloudApp

```diff
@@ -182,9 +183,30 @@ do
     PlistBuddy -c "Set com.apple.security.application-groups:0 ${APPGROUP}" "${ENTITLEMENTS[$i]}"
     PlistBuddy -c "Set OCAppIdentifierPrefix $APP_ID_PREFIX." "${LOCATIONS[$i]}/Info.plist"
     PlistBuddy -c "Set com.apple.developer.team-identifier ${TEAMID}" "${ENTITLEMENTS[$i]}"
+    
     if [[ "${ENTITLEMENTS[$i]}" =~ ^(ownCloud_File_Provider.entitlements)$ ]]; then
 	    PlistBuddy -c "Set NSExtension:NSExtensionFileProviderDocumentGroup $APPGROUP" "${LOCATIONS[$i]}/Info.plist"
-	fi
+	  fi 
+
+    if [[ "${ENTITLEMENTS[$i]}" =~ ^(ownCloud.entitlements)$ ]]; then
+        ## Add associated domains
+        if [ ! -z "$ASSOCIATED_DOMAINS" ]; then
+            PlistBuddy -c "Delete com.apple.developer.associated-domains array" "${ENTITLEMENTS[$i]}"  2>/dev/null
+            PlistBuddy -c "Add com.apple.developer.associated-domains array" "${ENTITLEMENTS[$i]}"  2>/dev/null
+            let COUNT=0;
+            IFS=',' read -ra strings <<< "$ASSOCIATED_DOMAINS";
+            for j in "${strings[@]}"
+            do
+                    echo "Adding associated domain: $j into entitlement: ${ENTITLEMENTS[$i]}"
+                    PlistBuddy -c "Add com.apple.developer.associated-domains:$COUNT string applinks:$j" "${ENTITLEMENTS[$i]}" 2>/dev/null
+                    ((COUNT++))
+                    if [[ $COUNT -eq 20 ]]; then
+                        echo "Can not add more then 20 associated-domains as per apple guide lines";
```

Comment:
Would be nice to add a colored output for that problem here, like:

`                         echo "${WARN}Can not add more then 20 associated-domains as per apple guide lines${NC}";`